app/templates/partials/widgets/relationship_widget.html

```diff
@@ -25,7 +25,8 @@
               "value": option.value,
               "name": answer.name,
               "id": option_id,
-              "checked": "checked" if option.selected
+              "checked": "checked" if option.selected,
+              "pattern": "[0-9]*"
```

Comment:
Not sure, @hamishtaplin should the relationship widget be changed as well? can you confirm if this is correct?